modules/server/tasks/bedrock.js

```diff
@@ -68,7 +69,8 @@ module.exports = function(grunt) {
     autoSettings.gruntDone = function (passed) {
       done(passed);
     };
-    autoSettings.stopOnFailure = options.stopOnFailure;
+    autoSettings.stopOnFailure = options.stopOnFailure || autoSettings.stopOnFailure;
```

Comment:
There is a Type module in server that can be used for ternaries on these things if we want to avoid `||` fallbacks.

Reply:
Right that would be harder. It's a bummer we can't use katamari on these things.